spec/features/allocation_information_spec.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "view an offender's allocation information" do
+  let!(:probation_officer_nomis_staff_id) { 485_636 }
+  let!(:nomis_offender_id_with_keyworker) { 'G4273GI' }
+  let!(:nomis_offender_id_without_keyworker) { 'G9403UP' }
+  let!(:allocated_at_tier) { 'A' }
+  let!(:prison) { 'LEI' }
+  let!(:pom_detail) {
+    PomDetail.create!(
+      nomis_staff_id: probation_officer_nomis_staff_id,
+      working_pattern: 1.0,
+      status: 'Active'
+    )
+  }
+
+  describe 'Offender has a key worker assigned', vcr: { cassette_name: :show_allocation_information_keyworker_assigned } do
+    before do
+      create_case_information_for(nomis_offender_id_with_keyworker)
+      create_allocation(nomis_offender_id_with_keyworker)
+    end
+
+    it "displays the Key Worker's details" do
+      signin_user
+
+      visit show_allocations_path(nomis_offender_id_with_keyworker)
+
+      expect(page).to have_css('h1', text: 'Allocation information')
+
+      # Prisoner
```

Comment:
I'm not sure whether I should be adding more assertions here (there are potentially many things to assert on this page)